src/components/tabs/_tabs.scss

```diff
@@ -111,7 +113,7 @@
 
     @include breakpoint(bp--sm--major) {
       background: transparent;
-      padding: 0.725rem 0 rem(10px);
+      padding: $spacing-sm 0 $spacing-sm;
```

Comment:
Just wanted to double-check if the presumable change in `padding` is intentional.

Reply:
Yes, I sat down with Anna on this one and we double checked everything looked ok.